src/utils/coordinates/coordinateExtractors.js

```diff
@@ -40,20 +39,20 @@ const webmercatorExtractor = (regexMatches) => {
     }
     let lon, lat
     if (regexMatches.length === 5) {
-        // 4 matches + global match, i.e. : (47)°(5.123)', (8)°(4.154)' (we inverse lat/lon to lon/lat in the process)
-        lon = Number(regexMatches[3]) + Number(regexMatches[4]) / 60.0
-        lat = Number(regexMatches[1]) + Number(regexMatches[2]) / 60.0
+        // 4 matches + global match, i.e. : (47)°(5.123)', (8)°(4.154)'
+        lon = Number(regexMatches[1]) + Number(regexMatches[2]) / 60.0
+        lat = Number(regexMatches[3]) + Number(regexMatches[4]) / 60.0
```

Comment:
For now it is ok but those extractors are not a good way as they are very dependent on the regex. I propose to rework this in a later PR to have the regex together with the extractor, this would make the code much easier to understand/read and less error prone

Reply:
I think this part of the app should be reworked too, yes.
It might have been one of the first thing I did while working on the project, and it pains my eyes to see some of my code from 4 years ago 🙈

Reply:
I've created a ticket to do that, after we've merged our work branch on `develop` https://jira.swisstopo.ch/browse/BGDIINF_SB-3165